examples/00-mapdl-examples/lathe_cutter.py

```diff
@@ -60,7 +60,8 @@
    Extended methods, and writing list to a file.
 
 7. **Advanced plotting:**
-   Use of `mesh.grid <ansys.mapdl.core.Mapdl.grid>`_ for additional post processing.
+   Use of `mesh.grid <https://mapdldocs.pyansys.com/mapdl_commands/graphics/_autosummary/ansys.mapdl.core.Mapdl.grid.html>`_
+   for additional post processing.
 
 
 Step 1: Launch and variables
```

Comment:
```suggestion
Step 1: Launch and add variables
```

Reply:
I'm not sure that "add" is the correct verb here--but it should have a verb. Perhaps "define" would be better?